CRM/Contribute/Selector/Search.php

```diff
@@ -366,6 +366,7 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL) {
     $allCampaigns = CRM_Campaign_BAO_Campaign::getCampaigns(NULL, NULL, FALSE, FALSE, FALSE, TRUE);
 
     while ($result->fetch()) {
+      $this->_query->convertToPseudoNames($result);
```

Comment:
Yay. Adds the contribution_status = 'Completed' to the array if contribution_status_id = 1 exists